sim/mcuboot-sys/build.rs

```diff
@@ -37,10 +38,23 @@ fn main() {
         conf.file("mbedtls/library/bignum.c");
         conf.file("mbedtls/library/asn1parse.c");
     } else if sig_ecdsa {
-        conf.define("MCUBOOT_SIGN_ECDSA", None);
+        conf.define("MCUBOOT_SIGN_EC256", None);
         conf.define("MCUBOOT_USE_TINYCRYPT", None);
-        // TODO: Compile files + tinycrypt.
-        panic!("ECDSA not yet implemented in sim");
+
+        conf.define("MBEDTLS_CONFIG_FILE", Some("<config-boot.h>"));
+        conf.include("mbedtls/include");
+        conf.include("../../ext/tinycrypt/lib/include");
+
+        conf.file("../../boot/zephyr/keys.c");
+
+        conf.file("../../ext/tinycrypt/lib/source/utils.c");
+        conf.file("../../ext/tinycrypt/lib/source/sha256.c");
+        conf.file("../../ext/tinycrypt/lib/source/ecc.c");
+        conf.file("../../ext/tinycrypt/lib/source/ecc_dsa.c");
+        conf.file("../../ext/tinycrypt/lib/source/ecc_platform_specific.c");
```

Comment:
If you're adding files from a new directory, you need to add a call to `walk_dir` below for that directory, or it will not see changes to those files as a dependency.

Reply:
fixed